message MessageHelloOK {
    required string sys_id = 1;
}
